doc/cli/npm-config.md

```diff
@@ -48,7 +48,7 @@ Echo the config value to stdout.
 
     npm config list
 
-Show all the config settings.
+Show all the config settings. `-l` show all settings including non-customized config pairs.
```

Comment:
I would probably reword this a bit to something like: `Use -l to also show defaults.`